client/components/folder-items.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export default class FolderItems extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ({
+      folders: []
+    });
+  }
+
+  componentDidMount() {
+    const token = window.localStorage.getItem('token');
+    fetch('/api/folders', {
+      headers: {
+        'X-Access-Token': token,
+        'Content-Type': 'application/json'
+      }
+    })
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          folders: data
+        });
+      })
+      .catch(err => console.error('Unexpected error occured', err));
+  }
+
+  render() {
+    const folders = this.state.folders;
+    const allFolders = folders.map(folder => (
+      <a href={`#home-page?folderId=${folder.folderId}`} key={folder.folderId}>
+        <div className='flex-basis' >
+          <div className='folders-container'>
+            <img src='https://upload.wikimedia.org/wikipedia/commons/thumb/5/59/OneDrive_Folder_Icon.svg/2048px-OneDrive_Folder_Icon.svg.png'></img>
```

Comment:
I recommend downloading this file and `src`ing it from your own images directory on the server.